phpstan-baseline.neon

```diff
@@ -8047,13 +8047,13 @@ parameters:
 			path: tests/lib/Server/Output/ValueObjectVisitor/OptionsTest.php
 
 		-
-			message: '#^Method PHPUnit\\Framework\\MockObject\\Builder\\InvocationMocker\<Ibexa\\Contracts\\Rest\\Output\\Visitor\>\:\:willReturnMap\(\) invoked with 2 parameters, 1 required\.$#'
+			message: '#^Method PHPUnit\\Framework\\MockObject\\Builder\\InvocationMocker\:\:willReturnMap\(\) invoked with 2 parameters, 1 required\.$#'
 			identifier: arguments.count
 			count: 1
 			path: tests/lib/Server/Output/ValueObjectVisitor/OptionsTest.php
 
 		-
-			message: '#^Parameter \#1 \$valueMap of method PHPUnit\\Framework\\MockObject\\Builder\\InvocationMocker\<Ibexa\\Contracts\\Rest\\Output\\Visitor\>\:\:willReturnMap\(\) expects array\<int, array\<int, mixed\>\>, array\<int, string\> given\.$#'
+			message: '#^Parameter \#1 \$valueMap of method PHPUnit\\Framework\\MockObject\\Builder\\InvocationMocker\:\:willReturnMap\(\) expects array\<int, array\<int, mixed\>\>, array\<int, string\> given\.$#'
```

Comment:
PHPStan reported errors are valid here.

Reply:
I did a rebase, and this code is already on main